sycl/test-e2e/SYCLBIN/lit.local.cfg

```diff
@@ -0,0 +1,7 @@
+syclbin_exec_opts = ""
+if 'target-spir' in config.available_features and 'ocloc' in config.available_features:
+  # Due to a bug in clang-linker-wrapper we cannot use -device * on Windows, so
```

Comment:
NIT: if the bug is reported in the open-source, can we add a link to it?

Reply:
Sure! It has been added.